commit d12207fb5a8cef131f262878f98da923af1f8fea
Author: Justus Winter <4winter@informatik.uni-hamburg.de>
Date:   Wed Apr 30 13:02:29 2014 +0200

    Install the mach_debug header files
    
    The task_set_name RPC introduced in 877a319c changed
    include/mach/gnumach.defs to include mach_debug/mach_debug_types.defs.
    Previously though, the debug headers were not installed.
    
    * Makefrag.am: Install the mach_debug header files.

diff --git a/Makefrag.am b/Makefrag.am
index c1387bd..d6dd77f 100644
--- a/Makefrag.am
+++ b/Makefrag.am
@@ -419,9 +419,18 @@ include_mach_eXec_HEADERS = \
 	include/mach/exec/elf.h \
 	include/mach/exec/exec.h
 
-# mach-debug-headers:= $(addprefix mach_debug/, hash_info.h ipc_info.h \
-# 		mach_debug.defs mach_debug_types.defs mach_debug_types.h \
-# 		pc_info.h vm_info.h slab_info.h)
+include_mach_debugdir = $(includedir)/mach_debug
+include_mach_debug_HEADERS = \
+	$(addprefix include/mach_debug/, \
+		hash_info.h \
+		ipc_info.h \
+		mach_debug.defs	\
+		mach_debug_types.defs \
+		mach_debug_types.h \
+		pc_info.h \
+		vm_info.h \
+		slab_info.h \
+	)
 
 # Other headers for the distribution.  We don't install these, because the
 # GNU C library has correct versions for users to use.
